app/controllers/gestionnaires/dossiers_controller.rb

```diff
@@ -109,8 +109,12 @@ def repasser_en_instruction
       if dossier.en_instruction?
         flash.notice = 'Le dossier est déjà en instruction.'
       else
-        flash.notice = "Le dossier #{dossier.id} a été repassé en instruction."
-        dossier.repasser_en_instruction!(current_gestionnaire)
+        if dossier.accepte? && !administration_signed_in?
+          flash.notice = 'Il n\'est pas possible de repasser un dossier accepte en instruction.'
```

Comment:
Apostrophe courbe (et accent)
```suggestion
          flash.notice = 'Il n’est pas possible de repasser un dossier accepté en instruction.'
```